src/main/java/uk/nhs/ctp/auditFinder/model/AuditValidationRequest.java

```diff
@@ -0,0 +1,12 @@
+package uk.nhs.ctp.auditFinder.model;
+
+import lombok.Data;
+
+@Data
+public class AuditValidationRequest {
+
+  private Long supplierInstanceId;
+  private String searchAuditId; // For service searches
```

Comment:
Although it's currently possible to determine which operation is being validated from which field is populated, I feel it might be cleaner to abstract the request to 'type' and 'id'. This may be necessary in future when we want to validate other operations which reuse the same IDs (e.g. encounter report)

Reply:
Yeah fair enough, just didn't seem necessary right now. We'll still be only populating one but I can add that extra type property.